src/query/execute_schema.c

```diff
@@ -10982,6 +10981,8 @@ build_attr_change_map (PARSER_CONTEXT * parser, DB_CTMPL * ctemplate, PT_NODE *
 	  chg_prop_idx = P_S_CONSTR_UNI;
 	  save_pt_costraint = true;
 	  break;
+	case PT_CONSTRAIN_NULL:
+	  attr_chg_properties->p[P_NOT_NULL] = ATT_CHG_PROPERTY_LOST;
```

Comment:
Can 'break' be omitted?

Reply:
it ommited break clause intentionally for applying common routine.

Reply:
I got it.